app/helpers/draws_helper.rb

```diff
@@ -14,7 +14,7 @@ def intent_deadline_str(draw)
   end
 
   def diff_class(diff)
-    raise ArgumentError unless diff.is_a? Integer
+    return 'negative' unless diff.is_a? Integer
```

Comment:
... I don't like this. I _am_ ok with returning `''` if we don't pass an integer.